binaryaudit/marinerABIdiff.py

```diff
@@ -0,0 +1,19 @@
+import RPM
+import run
+
+import sys
+
+def getRPMList(dirPath):
+    ret = run.runCommand(dirPath, "ls")
+    print "Return code: ", ret.returncode
+
+def groupRPMs(dirPath):
+    RPM.groupPackages(dirPath)
+
+def main():
+    dirPath = sys.argv[1]
+    getRPMList(dirPath)
+    groupRPMs(dirPath)
+
+if __name__ == "__main__":
+    main()
```

Comment:
This file seems to be an executable script. In this case, it should go into bin and added into `setup.py` to the scripts. Could you please rename it to `bin/ba_abidiff` (binary audit abidiff) similarly to the example that is already there. Perhaps there's a better idea for a convention, please let me know.

For the command help and argument parsing, the module `argparse` can be used:

https://docs.python.org/3/library/argparse.html

Thanks

Reply:
Perhaps an approach for this command could be like:

`ba_abidiff --json-out-file=/path/to/json/file --pkg-dir=/path/to/rpm/dir`

or

`ba_abidiff --json-out-file=/path/to/json/file /path/to/rpm/dir`

if `--pkg-dir` is ommited.

Thanks